src/lib/tls/asio/asio_stream.h

```diff
@@ -716,7 +716,7 @@ class Stream
                                 m_context.m_policy,
                                 m_context.m_rng,
                                 m_context.m_server_info,
-                                Protocol_Version::latest_tls_version()));
+                                Protocol_Version::TLS_V12));  // TODO don't hardcode
```

Comment:
- [ ] allow using asio stream (client) with TLS 1.3

Reply:
Just reverting this hunk was fine and resulted in a working asio wrapper.